README.md

```diff
@@ -1,3 +1,5 @@
 [![Build Status](https://api.travis-ci.org/MozillaSecurity/lithium.svg)](https://travis-ci.org/MozillaSecurity/lithium)
 [![Build Status](https://ci.appveyor.com/api/projects/status/69f2jn9f4kgjrffj/branch/master?svg=true)](https://ci.appveyor.com/project/MozillaSecurity/lithium/branch/master)
 [![codecov](https://codecov.io/gh/MozillaSecurity/lithium/branch/master/graph/badge.svg)](https://codecov.io/gh/MozillaSecurity/lithium)
+
+See [this page as a start](lithium/doc/using.html).
```

Comment:
It's hard to maintain 2 versions (using.html and README.md) though. Should we just choose one?

In any case, I'll remove the README.md change from this PR, you can follow up after that. :)